docs/source/examples/checks/distribution/trust_score_comparison.ipynb

```diff
@@ -2,106 +2,135 @@
  "cells": [
```

Comment:
<p>Really good notebook!!!!!!!!!! we should promote it to be in the front page of the docs.</p><p> </p><p>just 1 thought: I wonder whether a multi-class usecase will be interesting here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
it's an okay notebook, I've seen better...

Reply:
nothing beats the single feature contribution notebook.

About multiclass - it will look nice but will be a bit harder to show specific phenomena in the distribution graphs

Reply:
<p>:)</p>